src/renderer/index.js

```diff
@@ -1,6 +1,12 @@
 /* eslint-env browser */ /* global angular, staticPath */
 'use strict';
+import Logger from 'common/logger';
 const path = require('path');
+
+const log = new Logger('main');
```

Comment:
Shouldn't this be 'renderer'?

Reply:
Renderer is thread name, it will be added automatically by logger.
Besides that, there is the module name that will be different for each file / class / component / whatever that will allow us to track bugs easier.

In this case it will output `time level: RENDERER:main msg`